### PR TITLE
ROS#200: Demo commands

### PR DESCRIPTION
**Pull request by Glamhoth**

### CODE REVIEW

File: demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+use aerugo::{logln, RuntimeApi};
+
+#[derive(Copy, Clone, Debug)]
+pub enum OutputDataRate {
+    Invalid,
```
Comment (SteelPh0enix) [inline demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs:5]: that doesn't make sense, as "invalid" value is not a valid value :)
Potentially invalid value should be represented as Result, not actual enum value.

File: demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+use aerugo::{logln, RuntimeApi};
+
+#[derive(Copy, Clone, Debug)]
+pub enum GyroscopeScale {
+    Invalid,
```
Comment (SteelPh0enix) [inline demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs:5]: don't, see earlier comment

File: demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    Odr104,
+    Odr208,
+    Odr416,
+    Odr833,
+    Odr1666,
+    Odr3332,
+    Odr6664,
+}
+
+impl From<u8> for OutputDataRate {
```
Comment (SteelPh0enix) [inline demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs:18]: use TryFrom instead, see earlier comment

File: demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    Gs120,
+    Gs250,
+    Gs500,
+    Gs1000,
+    Gs2000,
+}
+
+impl From<u8> for GyroscopeScale {
```
Comment (SteelPh0enix) [inline demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs:13]: use TryFrom instead, see earlier comment

File: demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs (hunk 1 above)
Comment (Glamhoth) [reply demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs]: done

File: demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs (hunk 2 above)
Comment (Glamhoth) [reply demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs]: done

File: demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs (hunk 3 above)
Comment (Glamhoth) [reply demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs]: done

File: demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs (hunk 4 above)
Comment (Glamhoth) [reply demos/samv71-spi-accelerometer/src/task_set_gyroscope_scale.rs]: done

**Review by SteelPh0enix — APPROVED** [review]

**Pull request merged**